project-base/.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: bash
+dist: xenial
+stages:
+    - Linux
+    - MacOS
+services:
+    - docker
+os:
+    - linux
+jobs:
+    include:
+        -   stage: Linux
+            name: "Test linux install script"
+            script:
+                - echo 1 | ./scripts/install.sh
+                - docker-compose exec php-fpm ./phing checks tests-acceptance
+        -   stage: MacOS
+            name: "Test MacOS script in linux"
+            script:
+                - sudo apt install ruby
```

Comment:
is sudo necessary on Travis?

Reply:
I did not want to belive it either but yes it is necessary.

Travis output:
```
$ apt install ruby
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
The command "apt install ruby" exited with 100.
```